modules/multi_step/multi_step.tf

```diff
@@ -117,6 +117,24 @@ resource "datadog_synthetics_test" "multi" {
     monitor_options {
       renotify_interval = var.options_list.renotify_interval
     }
+
```

Comment:
I tested this with the next scenarios:

* not providing scheduling (no changes applied)
* providing the next scheduling:

```hcl
      scheduling = {
        timezone = "Europe/Madrid"
        timeframes = {
          monday    = { day = 1, from = "07:30", to = "21:30" }
          tuesday   = { day = 2, from = "07:30", to = "21:30" }
          wednesday = { day = 3, from = "07:30", to = "21:30" }
          thursday  = { day = 4, from = "07:30", to = "21:30" }
          friday    = { day = 5, from = "07:30", to = "21:30" }
          saturday  = { day = 6, from = "07:30", to = "21:30" }
          sunday    = { day = 7, from = "07:30", to = "21:30" }
        }
      }
```

result in the next test:
![image](https://github.com/ohpensource/terraform-datadog-ohp-synthetics/assets/97619732/68bc1ad0-825e-4185-b349-9098129e7408)